### PR TITLE
Distinguish between human/ML classifications when uploading

### PR DESCRIPTION
**Pull request by bfhealy**
This PR allows scope to take advantage of impending changes to skyportal (see https://github.com/skyportal/skyportal/pull/4441) that distinguish between human and ML classifications. It will be merged once the skyportal changes are deployed to Fritz.

### CODE REVIEW

File: doc/usage.md [hunk 1]
```diff
@@ -315,6 +315,8 @@ inputs:
 19. --result_filetag: name of tag appended to the result filename. Default is 'fritzUpload'.
 20. --result_format : result file format; one of csv, h5 or parquet. Default is parquet.
 21. --replace_classifications : flag to delete each source's existing classifications before posting new ones.
+22. --radius_arcsec: photometry search radius for uploaded sources.
+23. --no_ml: flag to post classifications that do not originate from an ML classifier.
```
Comment (mcoughlin) [inline doc/usage.md:319]: Is there a context where we are posting non-ML classifications through this pathway?

**Review by mcoughlin — APPROVED** [review]
Just one question!

File: doc/usage.md (hunk 1 above)
Comment (bfhealy) [reply doc/usage.md:319]: @mcoughlin I'd consider the initial training set classifications I posted to Fritz as distinct enough from subsequent ML classifications to warrant this flag.

Comment (mcoughlin) [reply doc/usage.md:319]: Ah got ya

**Pull request merged**